src/index.ts

```diff
@@ -2,18 +2,22 @@ import { sveltePreprocess } from './autoProcess';
 
 // default auto processor
 // crazy es6/cjs export mix for backward compatibility
-
+/** @deprecated Use the named export instead: `import { sveltePreprocess } from 'svelte-preprocess'` */
 // eslint-disable-next-line no-multi-assign
 export default exports = module.exports = sveltePreprocess;
```

Comment:
Do we still need this bizarre line? I don't quite remember why I had to do this whole exports/cjs/esm shenanigan. I guess it would be too early for another breaking change 😅

Reply:
yeah we need it because the default export is still present, and for interop reasons. I think it's best to not touch this line, just remove it in the next major.